src/main/java/org/opensearch/security/dlic/rest/api/RestApiAdminPrivilegesEvaluator.java

```diff
@@ -107,21 +111,24 @@ public boolean isCurrentUserRestApiAdminFor(final Endpoint endpoint, final Strin
             }
             return true;
         }
-        if (!ENDPOINTS_WITH_PERMISSIONS.containsKey(endpoint)) {
+        if (restapiAdminEnabled) {
```

Comment:
Nit; this function already has early returns, might as well use one to keep the workflow cleaner

```java
if (!restapiAdminEnabled) {
   return false;
}
```